bin/gen-pgbouncer-conf.sh

```diff
@@ -52,6 +52,7 @@ log_connections = ${PGBOUNCER_LOG_CONNECTIONS:-1}
 log_disconnections = ${PGBOUNCER_LOG_DISCONNECTIONS:-1}
 log_pooler_errors = ${PGBOUNCER_LOG_POOLER_ERRORS:-1}
 stats_period = ${PGBOUNCER_STATS_PERIOD:-60}
+stats_users = ${DB_USER}
```

Comment:
Good point - would need to split the uri out and pull the username earlier.
Alternative solution might be to allow a specific user to be configured for this via additional env vars.